app/scripts/modules/netflix/tableau/tableau.dataSource.ts

```diff
@@ -1,14 +1,15 @@
+import {module} from 'angular';
+
 import {DataSourceConfig} from 'core/application/service/applicationDataSource';
-import {APPLICATION_DATA_SOURCE_REGISTRY} from 'core/application/service/applicationDataSource.registry';
+import {
+  APPLICATION_DATA_SOURCE_REGISTRY,
+  ApplicationDataSourceRegistry
+} from 'core/application/service/applicationDataSource.registry';
```

Comment:
So this wrapping I actually like.  When you import too many things from the same file (or a few _long_ things) it gets impossible to read.  Did you guys decide what to do in this case?